backend/controller/identity/dal/dal.go

```diff
@@ -0,0 +1,42 @@
+package dal
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/TBD54566975/ftl/backend/controller/identity/dal/internal/sql"
+	"github.com/TBD54566975/ftl/backend/libdal"
+)
+
+type DAL struct {
+	*libdal.Handle[DAL]
+	db sql.Querier
+}
+
+func New(conn libdal.Connection) *DAL {
+	return &DAL{
+		db: sql.New(conn),
+		Handle: libdal.New(conn, func(h *libdal.Handle[DAL]) *DAL {
+			return &DAL{Handle: h, db: sql.New(h.Connection)}
+		}),
+	}
+}
+
+type EncryptedIdentity = sql.GetOnlyIdentityKeyRow
+
+func (d *DAL) GetOnlyIdentityKey(ctx context.Context) (*EncryptedIdentity, error) {
```

Comment:
Return a value here rather than a pointer so you're using the same kind consistently (eg. a value is used below).